controller/src/main/java/io/pravega/controller/server/rest/generated/api/ScopesApi.java

```diff
@@ -49,7 +49,7 @@ public ScopesApi(@Context ServletConfig servletContext) {
          String implClass = servletContext.getInitParameter("ScopesApi.implementation");
          if (implClass != null && !"".equals(implClass.trim())) {
             try {
-               delegate = (ScopesApiService) Class.forName(implClass).newInstance();
+               delegate = (ScopesApiService) Class.forName(implClass).getConstructor().newInstance();
```

Comment:
Was this re-generated? We shouldn't manually edit this.

Reply:
I see.
I found this committed, I did not notice it is generated stub.

Anyway the problem Class#newInstance has been deprecated so we cannot use it anymore.

I can look into a better solution.
Can you please point me to your workflow for generating these files ?
Usually generated files are not commited and they are generated during build.